build/BUILD.bazel

```
@@ -133,6 +139,25 @@ kong_directory_genrule(
         LN ${BUILD_DESTDIR}/openresty/bin/resty ${BUILD_DESTDIR}/bin/resty
         chmod -R "+rw" ${BUILD_DESTDIR}/openresty
 
+        CURL=${WORKSPACE_PATH}/$(echo '$(locations @curl//:curl)' | awk '{print $1}')
+        copy_with_filter ${CURL} ${BUILD_DESTDIR}/kong/
+        mkdir -p ${BUILD_DESTDIR}/kong/bin/
+        cp ${CURL}/bin/curl ${BUILD_DESTDIR}/kong/bin/
+        cp ${CURL}/bin/curl-config ${BUILD_DESTDIR}/kong/bin/
+        chmod -R "+rxw" ${BUILD_DESTDIR}/kong/bin
+
+        CACERTS=${WORKSPACE_PATH}/$(echo '$(locations @curl//:cacerts)' | awk '{print $1}')
+        mkdir -p ${BUILD_DESTDIR}/kong/ssl/
+        cp -r ${CACERTS} ${BUILD_DESTDIR}/kong/ssl/
+        (
+            cd ${BUILD_DESTDIR}/kong/ssl/
+            ln -sf cacerts-*.pem cacerts.pem
```

nit: for consistency consider using the `LN` function defined above

```suggestion
            LN cacerts-*.pem cacerts.pem
```

---

That's fair. Fixed.